.readthedocs.yml

```diff
@@ -8,3 +8,7 @@ mkdocs:
 python:
   install:
     - requirements: docs/requirements.txt
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.7"
```

Comment:
how do we decide what goes here?

Reply:
I think to avoid build failures.. ref: https://github.com/argoproj/argo-cd/pull/12956

Reply:
ya i meant did we pick those versions for a reason or does it not matter?